defaults/main.yml

```diff
@@ -119,3 +119,5 @@ steamengine_sf_log_directory: "log"
 steamengine_sf_cache_directory: "cache"
 # Php fpm version
 steamengine_sf_php_fpm_version: "php7.4-fpm"
+# List of systemd service to create for symfony
+steamengine_sf_systemd_services: []
```

Comment:
The fact that we are using systemd is irrelevant, this role gives abstraction to that, so let's just name this "steamengine_sf_services"